opm/simulators/timestepping/ConvergenceReport.hpp

```diff
@@ -51,6 +51,39 @@ namespace Opm
             Normal     = 1,
             TooLarge   = 2,
             NotANumber = 3,
+            ConvergenceMonitorFailure = 4,
```

Comment:
The severityOfWorstFailure() function expects severity of error to be worse for higher numbers. How severe is a failure due to the monitoring saying enough is enough? I would say greater than Normal (since we continue iterating for Normal) but less than the exploding residuals or NaNs, so put it at 2, resort and renumber? And maybe insert the comment I should have long ago:
```
// More severe problems should have higher numbers
```
or something similar.

Reply:
Good point, I agree. Fixed.